pkg/investigations/utils/alertmanager/alertmanager.go

```
@@ -0,0 +1,86 @@
+)
+
+// FiringAlert represents a single alert that is currently firing on a cluster.
+type FiringAlert struct {
+	Name     string
+	Severity string
+	State    string
+	Summary  string
+}
+
+type alertmanagerAlert struct {
+	Labels      map[string]string      `json:"labels"`
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)